<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeWarningDialog</class>
 <widget class="QDialog" name="SizeWarningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Large Files Detected</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Your backup contains large files</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="summaryFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="summaryLayout">
      <item>
       <widget class="QLabel" name="totalSizeIcon">
        <property name="text">
         <string>ðŸ“Š</string>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalSizeLabel">
        <property name="text">
         <string>Total backup size: 0 MB</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="fileCountLabel">
        <property name="text">
         <string>0 files analyzed</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="largeItemsLabel">
     <property name="text">
      <string>Large items (above warning threshold):</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="largeItemsTree">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Level</string>
      </property>
      <property name="width">
       <number>80</number>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
      <property name="width">
       <number>80</number>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
      <property name="width">
       <number>350</number>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Application</string>
      </property>
      <property name="width">
       <number>120</number>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="recommendationLabel">
     <property name="text">
      <string>Recommendation: Large files may exceed Git repository size limits. Consider adding exclusion patterns to .dfbuignore.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>color: gray; font-style: italic;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="criticalWarningLabel">
     <property name="text">
      <string></string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>color: red; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="continueBtn">
       <property name="text">
        <string>Continue Anyway</string>
       </property>
       <property name="toolTip">
        <string>Proceed with backup including large files</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="toolTip">
        <string>Cancel backup and return to main window</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
